micro-rdk-server/Cargo.toml

```diff
@@ -26,6 +26,7 @@ embedded-hal.workspace = true
 embedded-svc.workspace = true
 futures-lite.workspace = true
 micro-rdk = { workspace = true, features = ["esp32", "binstart", "provisioning"], default-features = true }
+micro-rdk-modular-driver-example = {path = "../examples/modular-drivers", features = ["esp32"]}
```

Comment:
`micro-rdk-modular-driver-example = {workspace = true, features = ["esp32"]}` results in an invalid workspace error